backend/src/main/java/ai/verta/modeldb/utils/ModelDBUtils.java

```diff
@@ -404,15 +406,34 @@ public static void scheduleTask(TimerTask task, int frequency, TimeUnit timeUnit
     executor.scheduleAtFixedRate(task, frequency, frequency, timeUnit);
   }
 
+  public static Throwable findRootCause(Throwable throwable) {
+    Objects.requireNonNull(throwable);
```

Comment:
why not do a simple null check?
This will throw a NPE if throwable is null . https://docs.oracle.com/javase/8/docs/api/java/util/Objects.html#requireNonNull-T-

Reply:
return null if parameter is null